Packs/Anomali_ThreatStream/Playbooks/playbook-Detonate_URL_-_ThreatStream.yml

```diff
@@ -60,7 +66,8 @@ tasks:
       submission_type:
         simple: url
       submission_value:
-        simple: ${inputs.URL}
+        complex:
+          root: inputs.URL
```

Comment:
pls add unique here as well - sorry missed that

Reply:
Added